02-household-queries/chainlit-household-bot.py

```diff
@@ -137,18 +175,72 @@ async def set_llm_model():
     await msg.stream_token(f"Done setting up {llm_name} LLM")
     await msg.send()
 
+async def set_embeddings():
+    settings = cl.user_session.get("settings")
+    llm_name = settings["model"]
+    embeddings = settings["embedding"]
+    msg = cl.Message(
+        author="backend",
+        content=f"Setting up LLM: {llm_name} with `{embeddings}`...\n",
```

Comment:
```suggestion
        content=f"Setting up embedding: `{embeddings}`...\n",
```
Also remove `llm_name` assignment on line 180.